### DISCUSSION TITLE
Replace Default Font with Gotham Font

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#14709] [General - Components / Directives / etc]
We want to replace default font wuth Gotham font completely because that is the branded fond of our organization.

[https://freefontsfamily.com/gotham-font-family/](https://freefontsfamily.com/gotham-font-family/)

I went through [https://quasar.dev/style/typography#introduction]( https://quasar.dev/style/typography#introduction) , but as per that approach we have to add class in each & every file which consume more time  , instead of that we want to replace the default font with Gotham Font.

Please guide me how to replace.

**Comment by ebeton-ro** [comment]
Why not use *{} to globally declare the font in app.scss? Set it with important flag and you should be good to go.

**Reply by jmriyaz84** [reply]
Sorry I didn't get you , can you please provide some working example.

**Reply by MilosPaunovic** [reply]
https://www.w3schools.com/cssref/sel_all.php
